### DISCUSSION TITLE
Cannot access the model in the prisma schema through the prisma client [nestjs]

### DISCUSSION THREAD

**Discussion by j-jmnz** [prisma/prisma#10168] [Q&A] [answered]
Hi! Is there any reason why the prisma client wouldn't give me access to a model I've declared in the prisma schema? I'm using prisma on nestjs.

I have 3 models in the schema (user, movie, entry), and in nest I can access the movie model by injecting the client into the service and calling `this.prisma.movie....` 
The weird thing is that I did exactly the same thing as I did with the movies service with the other two, but I can only access the movie model. I would really appreciate if anyone can help, I've been going a bit crazy over this :D

This is the link to the repo if it helps. [repo](https://github.com/j-jmnz/mi-db-api) The modules can be found in:
`src/movies
src/users
src/entries`

schema.prisma
```prisma // This is your Prisma schema file,
// learn more about it in the docs: https://pris.ly/d/prisma-schema

generator client {
  provider = "prisma-client-js"
}

generator nestgraphql {
  provider               = "node node_modules/prisma-nestjs-graphql"
  // output                 = "../src/@generated/prisma-nestjs-graphql"
  fields_Validator_from  = "class-validator"
  fields_Validator_input = true
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

model user {
  id       Int       @id @default(autoincrement())
  email    String    @unique
  username String
  password String
  avatar   String
  entries  entry[]
}

model entry {
  id      Int     @id @default(autoincrement())
  user    user?   @relation(fields: [userId], references: [id])
  userId  Int?
  type    String
  name    String
  comment String?
  rating  String?
}


model movie {
  id                   Int      @id @default(autoincrement())
  tmdb_id              Int
  backdrop_path        String?
  poster_path          String?
  genres               Json[]
  homepage             String?
  original_language    String?
  original_title       String?
  title                String
  overview             String
  release_date         String
  spoken_languages     Json[]
  production_countries Json[]
  cast                 Json[]
  crew                 Json[]
  status               String?
  created_at           DateTime @default(now())
  updated_at           DateTime @default(now()) @updatedAt
  tests                Json[]
  imdb_id              String
}

```

**Comment by j-jmnz** [comment, marked as answer]
Got help from the project's slack and figured out that issue was just an editror's issue. No more help is needed :D

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
